KdTree/KdTree/KdTree.h

```diff
@@ -18,46 +18,112 @@
 #ifndef _SEUTILS_KDTREE_H_
```

Comment:
It should have an Alexandria guard, right? Otherwise we can get problems when compiling?

Reply:
Good catch